paper-api/src/main/java/org/bukkit/attribute/Attribute.java

```diff
@@ -163,6 +163,12 @@ private static Attribute getAttribute(@NotNull String key) {
         return Registry.ATTRIBUTE.getOrThrow(NamespacedKey.minecraft(key));
     }
 
+    /**
+     * {@return the sentiment of this attribute}
```

Comment:
```suggestion
     * @return the sentiment of this attribute
```
i feel like this should also have a description

Reply:
The curly brackets around it already give it a description